glue/src/it/scala/com/gu/tableversions/glue/GlueMetastoreSpec.scala

```diff
@@ -0,0 +1,77 @@
+package com.gu.tableversions.glue
+
+import java.net.URI
+
+import cats.effect.IO
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{
+  AWSCredentialsProviderChain,
+  EnvironmentVariableCredentialsProvider,
+  InstanceProfileCredentialsProvider,
+  SystemPropertiesCredentialsProvider
+}
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.glue.model._
+import com.amazonaws.services.glue.{AWSGlue, AWSGlueClient}
+import com.gu.tableversions.core.Partition.PartitionColumn
+import com.gu.tableversions.core._
+import com.gu.tableversions.metastore.MetastoreSpec
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import cats.implicits._
+
+class GlueMetastoreSpec extends FlatSpec with Matchers with BeforeAndAfterAll with MetastoreSpec {
+
+  lazy val region = Regions.EU_WEST_1
+
+  lazy val credentials = new AWSCredentialsProviderChain(
+    new EnvironmentVariableCredentialsProvider(),
+    new SystemPropertiesCredentialsProvider(),
+    new ProfileCredentialsProvider("ophan"),
+    new ProfileCredentialsProvider(),
+    new InstanceProfileCredentialsProvider(false)
+  )
+  val glue: AWSGlue = AWSGlueClient.builder().withCredentials(credentials).withRegion(region).build()
+  val schema = "temp"
+  def tableLocation(tableName: String) = new URI(s"s3://ophan-temp-schema/test-data/$tableName")
+
+  val snapshotTable =
+    TableDefinition(TableName(schema, "test_snapshot"), tableLocation("test_snapshot"), PartitionSchema.snapshot)
+
+  val partitionedTable = TableDefinition(TableName(schema, "test_partitioned"),
+                                         tableLocation("test_partitioned"),
+                                         PartitionSchema(List(PartitionColumn("date"))))
```

Comment:
I'm just thinking that maybe I could add a random component to the table names so that multiple people running this test concurrently don't interfere with each other.

Reply:
Good idea!